api-tester/src/main/java/com/revenuecat/apitester/java/VirtualCurrencyInfoAPI.java

```diff
@@ -0,0 +1,12 @@
+package com.revenuecat.apitester.java;
+
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI;
+import com.revenuecat.purchases.VirtualCurrencyInfo;
+
+final class VirtualCurrencyInfoAPI {
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    static void check(final VirtualCurrencyInfo virtualCurrencyInfo) {
+        final long balance = virtualCurrencyInfo.getBalance();
+        final Long balance2 = virtualCurrencyInfo.getBalance();
```

Comment:
I'm not sure if it's really needed to have both the object and the primitive here... But I guess it doesn't hurt 👍

Reply:
Yeah, I'm not sure if it's completely necessary either, but since both are possible, I figured I'd add it 🤷‍♂️